table_engine/src/remote/model.rs

```diff
@@ -202,3 +240,48 @@ impl TryFrom<WriteRequest> for ceresdbproto::remote_engine::WriteRequest {
         })
     }
 }
+
+fn build_row_group_with_record_batch(
+    record_batches: Vec<arrow::record_batch::RecordBatch>,
+) -> Result<RowGroup> {
+    ensure!(
+        record_batches.len() == 1,
+        InvalidRecordBatchNumber {
+            batch_num: record_batches.len(),
+        }
+    );
```

Comment:
Just take one record batch as input parameter if only one batch is allowed in the `record_batches`.